components/generic/RelatedCollections.vue

```diff
@@ -29,50 +30,59 @@
     },
 
     props: {
-      query: {
+      title: {
         type: String,
         default: ''
+      },
+      relatedCollections: {
+        type: Array,
+        default: () => []
       }
     },
 
-    fetch() {
-      this.getSearchSuggestions(this.query);
-    },
-
-    data() {
-      return {
-        relatedCollections: []
-      };
-    },
-
     computed: {
       ...mapGetters({
         apiConfig: 'apis/config'
       })
     },
 
-    watch: {
-      query: '$fetch'
-    },
-
     methods: {
-      async getSearchSuggestions(query) {
-        this.relatedCollections = query === '' ? [] : await getEntitySuggestions(query, {
-          language: this.$i18n.locale,
-          rows: 4
-        });
-      },
+      linkGen(item) {
+        let id = '';
+        let name = '';
+
+        if (item.id) {
+          id = item.id;
+          name = item.prefLabel[this.$i18n.locale];
+        } else {
+          id = item.identifier;
+          name = item.name;
+        }
 
-      suggestionLinkGen(item) {
-        let id = item.id;
-        let name = item.prefLabel[this.$i18n.locale];
         const uriMatch = id.match(`^${this.apiConfig.data.origin}/([^/]+)(/base)?/(.+)$`);
         return this.$path({
           name: 'collections-type-all', params: {
             type: getEntityTypeHumanReadable(uriMatch[1]),
             pathMatch: getEntitySlug(id, name)
           }
         });
+      },
+      collectionTitle(item) {
+        if (item.prefLabel) {
+          return item.prefLabel[this.$i18n.locale];
+        } else {
+          return item.name;
+        }
+      },
+      imageUrl(item) {
+        if (typeof item.image === 'undefined' && typeof item.isShownBy === 'undefined') {
+          return '';
+        }
+        if (item.image) {
+          return item.image + '&size=w200';
+        } else if (item.isShownBy.thumbnail) {
+          return item.isShownBy.thumbnail + '&size=w200';
+        }
```

Comment:
The images are now squeezed, they are tiny, so hard to see. But perhaps adding `object-fit: cover;` on the badge img would improve the way the images are displayed.

Reply:
thanks I hadn't noticed since they're so tiny